packages/mobile/src/verify/VerificationEducationScreen.tsx

```diff
@@ -47,6 +60,12 @@ function VerificationEducationScreen({ route, navigation }: Props) {
   const numberVerified = useSelector(numberVerifiedSelector)
   const partOfOnboarding = !route.params?.hideOnboardingStep
 
+  const country = useMemo(() => {
+    const regionCode = getRegionCode(e164PhoneNumber || '')
+    const countries = new Countries(i18n.language)
```

Comment:
That was a great article, I've specially been guilty of `4. Using useMemo solely for referential equalities` in the past.

I didn't benchmark how expensive the call is tbh, but it's iterating and applying transformations over a list, if there are many re-renders it could get expensive I guess. I'll make sure not to abuse the use of `useMemo` :)